js/addressfinder/src/providers/nz.js

```diff
@@ -2,10 +2,19 @@ module.exports = {
     country: 'NZ',
     map(fullAddress, metaData) {
         let selected = new AddressFinder.NZSelectedAddress(fullAddress, metaData);
+        let street1Value, street2Value
```

Comment:
Probably best to keep this file consistent with semi-colons after each of these new lines.

Reply:
👍